character-sfx.js

```diff
@@ -97,6 +102,11 @@ class CharacterSfx {
       if (this.player.avatar.jumpState && !this.lastJumpState) {
         const audioSpec = soundFiles.jump[Math.floor(Math.random() * soundFiles.jump.length)];
         sounds.playSound(audioSpec);
+        
+        // play jump grunt 
+        if(this.player.hasAction('jump') && this.player.getAction('jump').trigger=='jump'){

```

Comment:
`===` instead of `==`.

Also incorrect spacing.

Reply:
Sorry https://github.com/webaverse/app/pull/2835/commits/242763dc83559799f39139499f6462586c314e8a